packages/blockchain-wallet-v4-frontend/src/scenes/Dex/Swap/ConfirmSwap/ConfirmSwap.tsx

```diff
@@ -17,15 +17,18 @@ import {
 } from '../components'
 import { Header } from './Header'
 
+const { DEX_SWAP_FORM } = model.components.dex
+
 type Props = {
-  formValues: DexSwapForm
   onClickBack: () => void
   walletCurrency: string
 }
 
-export const ConfirmSwap = ({ formValues, onClickBack, walletCurrency }: Props) => {
+export const ConfirmSwap = ({ onClickBack, walletCurrency }: Props) => {
   const dispatch = useDispatch()
 
+  const formValues = useSelector(selectors.form.getFormValues(DEX_SWAP_FORM)) as DexSwapForm
```

Comment:
Not a show-stopper, but you may want to rethink doing this. In the future if/when we remove redux-form this will be a pain to refactor.